schemas/session-replay/text-wireframe-schema.json

```diff
@@ -0,0 +1,133 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "text-wireframe-schema.json",
+  "title": "TextWireframe",
+  "type": "object",
+  "description": "Schema of all properties of a TextWireframe.",
+  "allOf": [
+    {
+      "$ref": "_common-shape-wireframe-schema.json"
+    },
+    {
+      "required": [
+        "text"
+      ],
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "The type of the wireframe.",
+          "const": "text",
+          "readOnly": true
+        },
+        "text": {
+          "type": "string",
+          "description": "The text value of the wireframe.",
+          "readOnly": false
+        },
+        "textStyle": {
+          "type": "object",
+          "description": "The style of this text field.",
+          "required": [
+            "family",
+            "font",
+            "color"
+          ],
+          "properties": {
+            "family": {
+              "type": "string",
+              "description": "The font family.",
+              "readOnly": true
+            },
+            "font": {
+              "type": "integer",
+              "description": "The font size.",
+              "readOnly": true
+            },
+            "color": {
+              "type": "string",
+              "description": "The font color as a string hexadecimal.",
+              "readOnly": true
+            }
+          },
+          "readOnly": true
+        },
+        "textPosition": {
+          "type": "object",
+          "description": "The position of the text in the wireframe.",
+          "required": [
+            "alignment",
+            "padding"
```

Comment:
Similar - both seem to have good defaults: `TL` + `0, 0, 0, 0` which will cover majority of use cases. IMO no reason to send it for every text wireframe.